### ISSUE TITLE
Sybil Report

### ISSUE THREAD

**Issue by YFalse** [connext/community-sybil-reports#58]
# Related Addresses
### Claster1 of 14 wallets
0x9bee3465119d4091421eefeedc66742dede98c1c 
0xad2000e9b460ae21fd70259f71b1a9e7884dabdd 
0x244eb136f986ee967a7986945b461cb271c5ac84
0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4 
0x47e6d8400009d1f1246907ed95fc18bf55f944f9
0x8e083605b21d8ab8ec70791462a22ea625eacf07
0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
0x6b365c3d4b3e34210c49f3a4789c3574921980da
0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
0xce099fca42f02018b5d25f0c5b83097bf7471efc
0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440
0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286
0xeb13593a73a5608c9fa7e4422a3126c33e3522a4 
0xd20e8021fdb3857fcf10a7fa7b0a8b6fd0578789 

### Claster2 of 7 wallets
0x5917d8ea043f6e2c76aa3304a1c2cdafac621642
0xe2c73b6a24e0f8aa8be519037f13bc738649326a
0xaa3c8541eece54c9199c112726f074ed7673e6cb
0x66233d37f71d900fd9f24531919b307a84e0adb2
0x1317e754e3b457be2d68dfcc38b95b93ac722a8e
0xb02d1525794107d5c8fd2e4e5a525583386fc5fb
0xcee24dbea4f151a2a1f0166eead59aca723c25fd

### Claster3 of 6 wallets
0xe21b9c6cd856ed4d4289ce8378dd266f12383805
0xa3b9b2014fd4bf1c9a5bcb921cb94fbd27342d3c
0xd0569404c94195a1730c4157b795e3e805c7f6dc
0x366b1eb1984a7f01799a1e2f389af62b4808ada7
0x2ecafe8980a14b86ec8af36fd4314e79ccf368bb
0xce3b79bac48359135e215ddd1c12f9067dab4575

### Claster4 of 11 wallets
0x34aededf3dbd039d2538b0d5ef4c379b9d1fcce4
0x211f7aa3d89b6f149cb583ae48a882ed4bc4118b
0x21e8fc9a596dcc8f6c1fe8fca732e4aec2e67739
0xe491c27ef574274d6958220078b187e5f5bc6a24
0x56694542414d0b02550262bef871c47e8c7daa4f
0xab567d20e147e41f0d4fe41a84415d6061fe2d83
0xd47abfca2bccb1e28197bbb7014538de32a4df4b
0x7bf2d125955155c25bc31bf65616aa911c2b2651
0xceed0fa6941a436f4cfeb0efa9a11e888547fb99
0xb8174030a920dadfe664feb96c743a9225de3ec3
0x6474dd2d8d1f926c94df34a415ba045f37d4ce51

### Claster5 of 4 wallets
0x4e3585dd1d4e11cb0d28c432d2e1ab5dcf0f4088
0x8bea66fefb1608a3cfecb3e7cd6ff56a513fbe8c
0xf05c9ed207be37b5b1a47a7a2a2d6e035799cca7
0x6b17d279dc4f544f7f3b93346827d5b51fa84b0c
### Claster6 of 42 wallets
0xe835d1e139398273707393250cc20a52ab449af3
0xf3b18c26a1b222c11e37f4fc9ef7ad084f98cedd
0xa0e5d2a0acd6e8a4e7bf562cc4f01d32dbc609b7
0xb9c8d182e937bf66df4400e23150a31592d9420e
0x2a5d78818182ae8a8ad44bdb5799967543ab6e62
0xbfa63ebec04bc7d7f4b4addccd613b524f065dcf
0xed5ce28e872ba8d204c88267d4122e33d51f57bb
0x788cc587a9cf3d46d51f75735901e7ea0e6c1c55
0x9d34f18539df5d0900ceb9a40f141b7dab459d4b
0x6912e9a73768fa376a2785504bfb8bf9e7d11fcc
0x7d93b56a1084eb7f36a8879e0ab45274378fcaaf
0xec8297ebadfcc1fdc981d7f1fb65e0c5ff8ddc70
0x3351027d27618d1047bc8f5282375e7162be629d
0x7bf915e4b94dc5226db5af7a41468223c2538d69
0x5358d4dc313ad1ce6f57faf36ee65f3d9dba908d
0x7d0b7c694bb5c017293a172dd5e1f2745385c82e
0xedea6da91e978e246f5c90aefdbe0ea124b67ca8
0x4e5b3a77c6de4a1d405dd95f83fef8a8b36ba7fc
0xb7bb389975895488ca380d9d98a9497a4e377bec
0xd42c9d4d4c860017e2b45ef7c424370abe1b3564
0x006a07961612a8916ed40b87eff4a1146447dbaa
0x0a8fcfefc82e368045ff30835c4b4201b84200b5
0x47b6e10844cbaf36661e8817e85c6484c378f363
0x11ff5ef2be52f28cfb2ece54eb2cc1806e0943e8
0x2bc8fa15a39105a6c673234f28317fa06ce95223
0x84b2fe0cb5f0e474d7bcdf53315722a909bd2d18
0xb0df5c9223cb12c6380dd3b3d7d51fd1b4d38cef
0xc940dbc051caf1776ec27bda923e1962895af625
0x1db46609a1ee10578329d82f113e7a2151d1d031
0x6e02182e3888de84327350d84d4e4adb89a18df0
0x9071b375c52a9da98a1824a9140cc8d50c9b9fb0
0x10946fe302436199a020bc7bbc6f5930d687ff7d
0x40e05f06db66af1cb5f57ed1f56426fc34d5fd88
0x668307b5de8302125e97539a9f55e6dad783bd0d
0xb46796da1d854ade9e6a548322f7bb7e39ea899e
0xabac8ebc08cdf93c3e9628886bb907d4693cb876
0x13d1e23baa4a28e6d07e4585f7018d71ab71252d
0xb611ba7069e8c1d28f98e93fd1641b3494fb3c81
0xec8447f61d5905831e306c9fc72ae3530ebf6691
0x3f9b0b55c625bba0140782299ec63f14150d2f00
0xcbc64c9945c3052925cf2c91fc431a75dbb90c1f
0x2be67b395d6e7a667b5eccbf923632f0736a6525
0x4300d499278870974b77d65027adb00a346cfbc1

# Reasoning
### Claster1: 
Wallet 0xad2000e9b460ae21fd70259f71b1a9e7884dabdd recieved funding on Arbitrum one chain (0.11eth and 10,971.6174 nextUSDC) between 6:17 pm UTC and 6:18 pm UTC on February 14th from the parent wallet 0x9bee3465119d4091421eefeedc66742dede98c1c. After funding between 6:22 pm UTC and 7:07 pm UTC on February 14th  transferred funds in equal amounts of 0.01 eth and 1100 nextusdc to the following addresses: 

1. 0x244eb136f986ee967a7986945b461cb271c5ac84
2. 0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4
3. 0x47e6d8400009d1f1246907ed95fc18bf55f944f9
4. 0x8e083605b21d8ab8ec70791462a22ea625eacf07
5. 0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
6. 0x6b365c3d4b3e34210c49f3a4789c3574921980da
7. 0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
8. 0xce099fca42f02018b5d25f0c5b83097bf7471efc
9. 0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440

On February 14th between 7:20 pm UTC and 7:35 pm UTC in the same sequence, the accounts added liquidity to the nextUSDC pool on Arbitrum One. At the next day on February 15th between 2:43 pm UTC and 3:25 pm UTC, the accounts removed all liquidity from the pool in equal parts (495.566 - 513.2301 nextUSDC and 576.2556 - 594.2322 USDC) and then transfered all the money from the accounts to the parent adress 0x9bee3465119d4091421eefeedc66742dede98c1c. 

Then the parent adress 0x9bee3465119d4091421eefeedc66742dede98c1c on February 15th 15:37 pm UTC sent 1100 nextUSDC on Arbitrum one chain to 0xeb13593a73a5608c9fa7e4422a3126c33e3522a4. 

0xeb13593a73a5608c9fa7e4422a3126c33e3522a4 added 1100 nextUSDC to the next pool on Arbitrum one. On March 10th between 10:00 pm UTC and 10:03 pm UTC removed all liquidity from the pool, swapped 510.6678 nextUSDC to 510.8353 USDC(Bridged) and sent them to the parent account 0x9bee3465119d4091421eefeedc66742dede98c1c. 
On March 16th between 5:01 pm UTC and 5:03 pm UTC parent account 0x9bee3465119d4091421eefeedc66742dede98c1c transfered 1205.1459 USDC(Bridged) and 0.003 eth on Arbitrum one chain to the 0xeb13593a73a5608c9fa7e4422a3126c33e3522a4. in the same window 0xeb13593a73a5608c9fa7e4422a3126c33e3522a4 added 1205.1459 USDC to the connext pool. On May 14th between 5:44 pm UTC and 5:46 pm UTC removed all liquidity from the pool (1204.974 USDC(Bridged) and sent 1204.974 USDC(Bridged) to the parent adress 0x9bee3465119d4091421eefeedc66742dede98c1c. 
On May 23rd between 12:02 am UTC and 12:55 am UTC received 0.004 eth and 4000 usdt on Arbitrum one from parent adress 0x9bee3465119d4091421eefeedc66742dede98c1c and added 4000 usdt to the connext usdt pool 
![image](https://github.com/connext/community-sybil-reports/assets/85909114/7d5d35f7-6585-4ccb-8025-97308ee695c7)

0x9bee3465119d4091421eefeedc66742dede98c1c, 0xeb13593a73a5608c9fa7e4422a3126c33e3522a4, 0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286,  using the same binance,bybit deposit adress at the same time "0xc74b688c72a926d70ec7c8b87dd38decbb59a138", bybit adress "0xb3553afafaf9729f1338926efad471515466da16".
Deposited on September 5th 2022 between 7:33 pm UTC and 7:34 pm UTC to the 0xc74b688c72a926d70ec7c8b87dd38decbb59a138. 

Wallet 0xd20e8021fdb3857fcf10a7fa7b0a8b6fd0578789 Deposited on June 24th at 1:04 pm UTC to the 0xc74b688c72a926d70ec7c8b87dd38decbb59a138
0xd20e8021fdb3857fcf10a7fa7b0a8b6fd0578789 All transactions are connected with connext.
received 0.1 matic at Feb 16 at 2:19 pm UTC from 0x9bee3465119d4091421eefeedc66742dede98c1c
received 0.002 eth on arb network Feb 16 at 2:20 pm UTC from 0x9bee3465119d4091421eefeedc66742dede98c1c
withdrew 999.1083 usdc to binance in the polygon network on February 16 at 2:36 pm UTC
received 0.17 matic on February 16 at 2:53 pm UTC from 0x9bee3465119d4091421eefeedc66742dede98c1c to pay fees, then between 2:54 pm UTC and 2:56 pm UTC exchanged 999.1083 usdc for 996.4512 nextUSDC in polygon networks, and then added them to the liquidity pool.

On March 11 at 10:19 pm UTC exchanged 1022.3457 nextUSDC for 1020.3663 usdc, and then immediately withdraw to binance adress "0xc74b688c72a926d70ec7c8b87dd38decbb59a138"
On May 15, from 9:29 am UTC to 9:30 am UTC, received 0.001 eth on the Arbitrum one network from 0x9bee3465119d4091421eefeedc66742dede98c1c, transferred 1198.1664 usdc(Bridged) from the arbitrum network to the bnb network via the connext bridge and sent 0.001 eth back to parent wallet 0x9bee3465119d4091421eefeedc66742dede98c1c

On May 25, between 1:20 pm UTC and 2:28 pm UTC, I received 8200.004 usdt in the bnb network from 0x9bee3465119d4091421eefeedc66742dede98c1c, transferred them equally to the network: arbiter and polygon, and added them to the connext liquidity pools.

On June 24, between 12:45 pm UTC and 13:04 pm UTC, withdrawed 7355.1149 usdt of liquidity from connext in the Arbitrum one network and transferred 7355.1149 usdt to the binance address 0xc74b688c72a926d70ec7c8b87dd38decbb59a138
### Claster2
Same transactions on same networks on wallets with same dates and no other activity except connext network. Using same cex deposit adress to withdraw funds from the wallets. Wallets  are controlled by the same person.
![image](https://github.com/connext/community-sybil-reports/assets/85909114/bbf0a5c9-fd55-4699-81c2-054366857b9a)
### Claster 3
All wallets are using the same cex adress at the same time and same wallet activity.
### Claster4
All addresses are using the same cex adress for deposit and all of the deposits to cex were done at the same time.
Same wallet creation date.
### Claster5
All addresses are using the same cex adress for deposit and all of the deposits to cex were done at the same time.
Same wallet creation date.
### Claster6
All addresses received funds from the same wallets in list. All wallets were created 700-650 days ago
Transactions on these wallets are identical, they were made at the same time in the same networks, mainly in the gnosis chain, polygon, fantom, iotex, bnb.
After all transactions were made, the money was transferred to another wallet from this list, the balance of these wallets is on average $1-5.

# Methodology
### Claster1
All wallets are related to the parent wallet "0x9bee3465119d4091421eefeedc66742dede98c1c ". Parent wallet funding them, and after the activities were completed, the funds were transferred back to it.
The transactions on the wallets are the same and completed at the same time, and the same exchange addresses are used to deposit to cex acccount. 
All the data was collected by analyzing and viewing the transactions of the parent wallet.
 https://debank.com/profile/0x9bee3465119d4091421eefeedc66742dede98c1c
 The resulting addresses were analyzed to find repeating transfers at standard intervals and sequence and based on this infromation this list of sybils was created. 

![image](https://github.com/connext/community-sybil-reports/assets/85909114/1a05efe0-1cb6-4e2c-a514-a0e7aa6803ca)

### Claster2
I used debank to check first wallet activity and find out that there is only activity in connext network and money gone to another wallet that was created the same time as first.
Second account sent money to the okex deposit adress, then I checked it and found out that A bunch of same wallets using the same cex wallet for deposit to okex.
Based on the above facts, we can understand that these wallets were created in order to receive a drop and are controlled by the same person.

wallet "0x5917d8ea043f6e2c76aa3304a1c2cdafac621642"
On March 16th between 12:40 pm UTC and 12:51 pm UTC received 313.3315 usdc on polygon and bridged to gnosis chain, then added 301.7546 usdc to the liquidity pool on gnosis chain.
On April 1st between 12:21 pm UTC and 12:25 pm UTC removed liquidity and send all of the money to the 0xe2c73b6a24e0f8aa8be519037f13bc738649326a wallet.

wallet "0xe2c73b6a24e0f8aa8be519037f13bc738649326a"
On March 16th between 12:40 pm UTC and 12:49 pm UTC received 312.5 usdc on polygon, bridged to gnosis chain and added liquidity to the gnosis pool on connext. On April 1st between 12:20 pm UTC and 12:30 pm UTC removed all liquidity, received 614.5696 usdc on gnosis chain from 0x5917d8ea043f6e2c76aa3304a1c2cdafac621642 and sent all of the money to okex.
All wallets related to the same cex address "0xfc70295492cd10add77d27caf7a2f9e8a623fe51"
https://debank.com/profile/0xfc70295492cd10add77d27caf7a2f9e8a623fe51/history?chain=matic&token=0x2791bca1f2de4661ed88a30c99a7a9449aa84174_USDC
![image](https://github.com/connext/community-sybil-reports/assets/85909114/2cca6eef-c515-4a25-a60e-3c4507b48562)
### Claster 3
They are using same cex deposit adress to withdraw funds from wallets.
https://debank.com/profile/0x8de7e875b12e75b834b897eb7ae8f8de695b3a94/history?chain=arb&token=0x912ce59144191c1204e64559fe8253a0e49e6548_ARB
There you can check that they are related to the same cex adress: 0x8de7e875b12e75b834b897eb7ae8f8de695b3a94
![image](https://github.com/connext/community-sybil-reports/assets/85909114/528b4715-e966-4068-9f72-0e08210f4e39)

![image](https://github.com/connext/community-sybil-reports/assets/85909114/0c9eaf4a-48d4-41b0-b4b3-012bf08f3ec7)
### Claster4
I used debank to check who deposited to the cex adress and find out that there are 11 wallets related to the same cex adress and deposited at the same time then I check all of the wallets and they were created at the same dates with same activity. That's why they're controlled by the same person.
Binance deposit adress: 0xe73e0a59c5789fd4ffc73ccc7fd61cdb4738639e
https://debank.com/profile/0xe73e0a59c5789fd4ffc73ccc7fd61cdb4738639e/history?chain=arb
There you can check for incoming transactions on this cex address
![image](https://github.com/connext/community-sybil-reports/assets/85909114/a6503344-8410-42b4-aa5c-228208970800)
### Claster5
wallets are using the same cex adress for deposit.
gate 0x8d63c6dd1c6261cec0b0dd3b890e87810cabbb46
Binance 0x994b27ed86cc2357a5c5e3b86372a0eedc4fa599
![image](https://github.com/connext/community-sybil-reports/assets/85909114/a2262304-291f-42c0-a6fe-4ec2c0c6aded)
### Claster6
I used debank and arkham to analyze the transactions of each wallet from this list, analyzing by date of creation, parent wallet, where the funds went, by transactions and networks - these wallets are controlled by the same person
![image](https://github.com/connext/community-sybil-reports/assets/85909114/c1b3346d-2bd0-4f3c-adb5-cf1d6d96d633)

Just for example. First 15 wallets from this list.
1. Those wallets like all wallets from this list are using the same protocols on chains at the same dates. Protocols such as: 0x; Paraswap, Spookyswap, Sushiswap, Gelato, MimoDefi, Pancakeswap, 1Inch.
2. There are the same txn's volumes on protocols on every wallet.
3. Using same tokens to trade for.
4. They were created at the same dates and all related to each other.
5. Also they're balances are equal 1-5$.
In conclusion I wanna say that based on this and Arkham information they are all controlled by the same person.
![image](https://github.com/connext/community-sybil-reports/assets/85909114/f63f86c6-c498-4d59-8da2-dcc1baa8077d)

# Rewards Address

0x764818f233Aae9b3d2d93bEfC649a3A24A526B51

**Comment by YFalse** [comment]
If you need any links or more proofs then let me know

**Comment by ArjunBhuptani** [comment]
Cluster 1:

Invalid. There's definitely suspicious behavior between these wallets, but not all of them interacted with Connext consistently. There's a likelihood that there is a sybil cluster related to this out there but it's also possible that some of the addresses in this list are not sybils.

**Comment by YFalse** [comment]
> * 0x244eb136f986ee967a7986945b461cb271c5ac84
> * 0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4
> * 0x47e6d8400009d1f1246907ed95fc18bf55f944f9
> * 0x8e083605b21d8ab8ec70791462a22ea625eacf07
> * 0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
> * 0x6b365c3d4b3e34210c49f3a4789c3574921980da
> * 0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
> * 0xce099fca42f02018b5d25f0c5b83097bf7471efc
> * 0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440

Hello. Let me explain.  Parent addres "0x9bee3465119d4091421eefeedc66742dede98c1c" sent money to the wallet "0xad2000e9b460ae21fd70259f71b1a9e7884dabdd" that interacted only with connext on arbitrum and and was used as an interlayer between those wallets. They were created at the same date, same trascantions at the same time (Only connext) on the same network and at the end the money was sent back to the parent wallet.
0x244eb136f986ee967a7986945b461cb271c5ac84
0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4
0x47e6d8400009d1f1246907ed95fc18bf55f944f9
0x8e083605b21d8ab8ec70791462a22ea625eacf07
0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
0x6b365c3d4b3e34210c49f3a4789c3574921980da
0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
0xce099fca42f02018b5d25f0c5b83097bf7471efc
0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440

**Comment by YFalse** [comment]
> > * 0x244eb136f986ee967a7986945b461cb271c5ac84
> > * 0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4
> > * 0x47e6d8400009d1f1246907ed95fc18bf55f944f9
> > * 0x8e083605b21d8ab8ec70791462a22ea625eacf07
> > * 0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
> > * 0x6b365c3d4b3e34210c49f3a4789c3574921980da
> > * 0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
> > * 0xce099fca42f02018b5d25f0c5b83097bf7471efc
> > * 0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440
> 
> Hello. Let me explain. Parent addres "0x9bee3465119d4091421eefeedc66742dede98c1c" sent money to the wallet "0xad2000e9b460ae21fd70259f71b1a9e7884dabdd" that interacted only with connext on arbitrum and and was used as an interlayer between those wallets. They were created at the same date, same trascantions at the same time (Only connext) on the same network and at the end the money was sent back to the parent wallet. 0x244eb136f986ee967a7986945b461cb271c5ac84 0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4 0x47e6d8400009d1f1246907ed95fc18bf55f944f9 0x8e083605b21d8ab8ec70791462a22ea625eacf07 0x47778dc2da9ed23e8a148bb55e5ef12039d8a139 0x6b365c3d4b3e34210c49f3a4789c3574921980da 0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62 0xce099fca42f02018b5d25f0c5b83097bf7471efc 0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440

As soon as the wallets were funded from the layer wallet, they began to add liquidity in the order in which they received the funds

**Comment by YFalse** [comment]
Wallets: "0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286", "0xeb13593a73a5608c9fa7e4422a3126c33e3522a4" using the same cex deposit addres as the parent wallet. If we go through activity of this wallets on chain: "0x9bee3465119d4091421eefeedc66742dede98c1c"(parent wallet) , "0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286", "0xeb13593a73a5608c9fa7e4422a3126c33e3522a4" then we find out that they are doing exactly same activities on different chains a couple of minutes apart

**Comment by ArjunBhuptani** [comment]
Cluster 2 - Valid:
- Clear pattern of farming connext
- Clear pattern of funding addresses from CEX

**Comment by ArjunBhuptani** [comment]
Cluster 3 - Valid:
- Addresses are clearly related: 
https://platform.arkhamintelligence.com/visualizer/entity/0xE21b9c6CD856ed4d4289ce8378dD266F12383805,0xA3B9B2014fd4bF1C9A5bCb921cb94FBd27342d3c,0xd0569404C94195A1730C4157B795e3E805C7F6dC,0x366b1EB1984A7f01799A1E2f389aF62b4808ada7,0x2EcaFE8980A14b86ec8aF36Fd4314e79Ccf368bB,0xcE3B79BaC48359135E215dDd1C12F9067DAB4575?flow=self&from=connext&positions=%7B%220xd0569404C94195A1730C4157B795e3E805C7F6dC%22%3A%7B%22fx%22%3A-37%2C%22fy%22%3A-59%7D%2C%220xE21b9c6CD856ed4d4289ce8378dD266F12383805%22%3A%7B%22fx%22%3A-25%2C%22fy%22%3A34%7D%2C%220xA3B9B2014fd4bF1C9A5bCb921cb94FBd27342d3c%22%3A%7B%22fx%22%3A-82%2C%22fy%22%3A25%7D%2C%220x2EcaFE8980A14b86ec8aF36Fd4314e79Ccf368bB%22%3A%7B%22fx%22%3A-13%2C%22fy%22%3A-45%7D%2C%22binance%22%3A%7B%22fx%22%3A-92%2C%22fy%22%3A30%7D%2C%22okx%22%3A%7B%22fx%22%3A-59%2C%22fy%22%3A20%7D%2C%22connext%22%3A%7B%22fx%22%3A-97%2C%22fy%22%3A-62%7D%2C%220xcE3B79BaC48359135E215dDd1C12F9067DAB4575%22%3A%7B%22fx%22%3A-122%2C%22fy%22%3A-5%7D%2C%220x366b1EB1984A7f01799A1E2f389aF62b4808ada7%22%3A%7B%22fx%22%3A-8%2C%22fy%22%3A-12%7D%2C%22bitkeep%22%3A%7B%22fx%22%3A-104%2C%22fy%22%3A-68%7D%7D&sortDir=desc&sortKey=time&timeGte=1655942400000&to=connext&usdGte=0.1
- There isn't a ton of Connext farming, but it's clearly happening in short bursts

**Comment by ArjunBhuptani** [comment]
Cluster 4:

Valid. Note that this isn't the full cluster - #404 reported some additional addresses not included here and we will need to merge lists.

- Addresses clearly related: 
https://platform.arkhamintelligence.com/visualizer/entity/0x34AEdedF3dBd039d2538B0D5eF4c379b9D1FCCe4,0x211f7aA3D89b6f149Cb583aE48a882Ed4bC4118B,0x21E8fc9a596DcC8F6c1FE8FCa732e4Aec2E67739,0xE491c27Ef574274d6958220078b187E5F5BC6A24,0x56694542414d0b02550262bEf871C47E8C7DAA4f,0xAB567D20e147E41f0d4fe41a84415D6061Fe2d83,0xD47aBfCa2bccB1E28197bbb7014538De32A4DF4B,0x7BF2D125955155c25bC31bF65616aa911c2b2651,0xcEed0fA6941A436F4cFEB0efa9A11E888547fb99,0xB8174030A920dADfe664feB96c743A9225de3Ec3,0x6474Dd2d8d1f926C94dF34A415ba045F37D4ce51?flow=self&positions=%7B%220x56694542414d0b02550262bEf871C47E8C7DAA4f%22%3A%7B%22fx%22%3A24%2C%22fy%22%3A3%7D%7D&sortDir=desc&sortKey=time&usdGte=0.1
- Clear farming activity of Connext:
https://platform.arkhamintelligence.com/visualizer/entity/0x34AEdedF3dBd039d2538B0D5eF4c379b9D1FCCe4,0x211f7aA3D89b6f149Cb583aE48a882Ed4bC4118B,0x21E8fc9a596DcC8F6c1FE8FCa732e4Aec2E67739,0xE491c27Ef574274d6958220078b187E5F5BC6A24,0x56694542414d0b02550262bEf871C47E8C7DAA4f,0xAB567D20e147E41f0d4fe41a84415D6061Fe2d83,0xD47aBfCa2bccB1E28197bbb7014538De32A4DF4B,0x7BF2D125955155c25bC31bF65616aa911c2b2651,0xcEed0fA6941A436F4cFEB0efa9A11E888547fb99,0xB8174030A920dADfe664feB96c743A9225de3Ec3,0x6474Dd2d8d1f926C94dF34A415ba045F37D4ce51?flow=all&from=connext&positions=%7B%7D&sortDir=desc&sortKey=time&timeGte=1636588800000&timeLte=1638662400000&usdGte=0.1

**Comment by ArjunBhuptani** [comment]
Cluster 5:

Invalid. Too great of a chance that this is valid user activity for a cluster of this size.

**Comment by ArjunBhuptani** [comment]
Cluster 6:

Valid.

- Definitely related to each other:
https://platform.arkhamintelligence.com/visualizer/entity/0xe835D1e139398273707393250CC20a52AB449Af3,0xF3b18C26A1b222C11e37f4fC9ef7AD084F98CEdD,0xA0E5d2a0Acd6e8a4e7BF562cC4f01d32DBC609B7,0xB9C8D182e937bF66dF4400E23150a31592d9420E,0x2a5D78818182AE8a8aD44bdb5799967543Ab6e62,0xbfA63ebec04bc7D7F4b4aDDcCd613b524f065dcf,0xEd5ce28e872BA8d204c88267D4122E33D51f57bB,0x788CC587A9cf3d46D51f75735901E7Ea0e6c1c55,0x9d34f18539Df5D0900CeB9A40F141B7dAB459D4b,0x6912E9a73768fA376A2785504BFb8bF9E7d11fcc,0x7d93b56a1084eb7f36a8879e0Ab45274378fCaAf,0xEC8297EbaDfCc1Fdc981D7f1fB65E0c5ff8dDc70,0x3351027d27618d1047bC8F5282375E7162be629d,0x7bf915e4B94Dc5226DB5af7A41468223C2538D69,0x5358d4dC313aD1cE6f57Faf36eE65f3D9dbA908D,0x7D0b7C694BB5c017293a172DD5e1F2745385c82E,0xEDEA6Da91E978e246f5c90AEFdbE0EA124B67CA8,0x4e5b3A77C6de4A1d405dD95F83feF8a8b36BA7Fc,0xb7BB389975895488Ca380d9d98A9497a4e377bEc,0xD42c9d4D4c860017e2B45Ef7c424370aBE1B3564,0x006a07961612a8916ed40B87eFF4A1146447dBAA,0x10946fE302436199a020bC7bbC6f5930d687fF7D,0x668307B5DE8302125e97539a9f55e6dAd783bD0D,0x13d1e23Baa4A28e6d07E4585f7018D71aB71252D,0x3F9B0B55c625BbA0140782299ec63f14150d2f00?flow=all&positions=%7B%220xbfA63ebec04bc7D7F4b4aDDcCd613b524f065dcf%22%3A%7B%22fx%22%3A-137%2C%22fy%22%3A20%7D%7D&sortDir=desc&sortKey=time&usdGte=0.1
- There isn't a super consistent pattern of farming Connext, but it's pretty clear all these wallets have been funded by the 0x7d address via CEXs

**Comment by YFalse** [comment]
@ArjunBhuptani What about this wallets from cluster1:
0xad2000e9b460ae21fd70259f71b1a9e7884dabdd
0x244eb136f986ee967a7986945b461cb271c5ac84
0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4
0x47e6d8400009d1f1246907ed95fc18bf55f944f9
0x8e083605b21d8ab8ec70791462a22ea625eacf07
0x47778dc2da9ed23e8a148bb55e5ef12039d8a139
0x6b365c3d4b3e34210c49f3a4789c3574921980da
0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62
0xce099fca42f02018b5d25f0c5b83097bf7471efc
0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440

**Comment by Stakeridoo** [comment]
@ArjunBhuptani I think Group 1 is also legit, they are all on the eligible list.
You can see they have a lot of transactions in between and as @YFalse said the Wallet [0xad2000e9b460ae21fd70259f71b1a9e7884dabdd](https://debank.com/profile/0xad2000e9b460ae21fd70259f71b1a9e7884dabdd/history?t=1693684140719&r=48416) distributed funds and also other [connections](https://dune.com/stakeridoo/sybilconnext?addresses_td63d9=0x9bee3465119d4091421eefeedc66742dede98c1c%2C+0xad2000e9b460ae21fd70259f71b1a9e7884dabdd%2C+0x244eb136f986ee967a7986945b461cb271c5ac84%2C+0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4%2C+0x47e6d8400009d1f1246907ed95fc18bf55f944f9%2C+0x8e083605b21d8ab8ec70791462a22ea625eacf07%2C+0x47778dc2da9ed23e8a148bb55e5ef12039d8a139%2C+0x6b365c3d4b3e34210c49f3a4789c3574921980da%2C+0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62%2C+0xce099fca42f02018b5d25f0c5b83097bf7471efc%2C+0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440%2C+0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286%2C+0xeb13593a73a5608c9fa7e4422a3126c33e3522a4%2C+0xd20e8021fdb3857fcf10a7fa7b0a8b6fd0578789) to all those mentioned wallet's can be found.
Also the [Transaction amount and first transaction date](https://dune.com/stakeridoo/sybil-connext?addresses_tf1ab9=0x9bee3465119d4091421eefeedc66742dede98c1c%2C+0xad2000e9b460ae21fd70259f71b1a9e7884dabdd%2C+0x244eb136f986ee967a7986945b461cb271c5ac84%2C+0x4e7bda4869ebe9fa42ad254ea529fb7233a08cb4%2C+0x47e6d8400009d1f1246907ed95fc18bf55f944f9%2C+0x8e083605b21d8ab8ec70791462a22ea625eacf07%2C+0x47778dc2da9ed23e8a148bb55e5ef12039d8a139%2C+0x6b365c3d4b3e34210c49f3a4789c3574921980da%2C+0xfd0cb35f3fcbd37544f003eb5a9d9a2f12059c62%2C+0xce099fca42f02018b5d25f0c5b83097bf7471efc%2C+0xdd5ec47d6f893c316fb1476fbdcf7b7b68db8440%2C+0xc6e64f2da8c0c76926bf92fb1bd9dff19837b286%2C+0xeb13593a73a5608c9fa7e4422a3126c33e3522a4%2C+0xd20e8021fdb3857fcf10a7fa7b0a8b6fd0578789) for 10 Wallet's is the same (2023-02-14 on Arbitrum, 9 from them have extactly 7 transactions), the other wallets have nearly same first transaction on Celo, Polygon or Gnosis

**Comment by A-Keyboard** [comment]
@YFalse Can you dm me on Discord, my discord is khlilo

**Comment by YFalse** [comment]
> khlilo

Done

**Comment by YFalse** [comment]
> @YFalse Can you dm me on Discord, my discord is khlilo

@A-Keyboard 686784
